### PR TITLE
Set strict timestamp to false in zip file

### PR DESCRIPTION
**Pull request by eduardomourar**
*Issue #, if available:*

*Description of changes:*

Writing to a zip file will fail with the following message whenever the file have an old modified timestamp:
```
ZIP does not support timestamps before 1980
```

To avoid that, we will use a new argument called `strict_timestamps ` introduced to the [ZipFile library](https://docs.python.org/3.8/library/zipfile.html#zipfile-objects) in Python 3.8.

This is related to the following issue with Node.js dependencies: https://github.com/aws/aws-cli/issues/2639.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by eduardomourar** [comment]
The reproduced error can be checked here: https://travis-ci.com/github/aws-cloudformation/cloudformation-cli/jobs/397905571

File: setup.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ def find_version(*file_paths):
         "hypothesis>=4.32",
         "colorama>=0.4.1",
         "docker>=4.3.1",
+        "zipfile38>=0.0.2,<0.2",
```
[the remark this answers is not in the document]
Comment (eduardomourar) [reply setup.py:53]: I will ask the developer, but I see python software foundation license here: https://pypi.org/project/zipfile38/#data

**Comment by RichieRunner** [comment]
can confirm that is a problem. is this going to be resolved?

**Comment by hsiaoa** [comment]
having the same issue. please resolve, thanks.

File: setup.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (eduardomourar) [reply setup.py:53]: i have created an issue in the repo, but no response yet: https://github.com/lorencarvalho/zipfile38/issues/1

[the remark this answers is not in the document]
Comment (eduardomourar) [reply setup.py]: author has added the license and there is a new version available

**Pull request closed without merging**